src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -222,7 +222,7 @@ public partial class SslStream : System.Net.Security.AuthenticatedStream
         public void Write(byte[] buffer) { }
         public override void Write(byte[] buffer, int offset, int count) { }
         public override System.Threading.Tasks.Task WriteAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
-        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Was this change intended?

Reply:
Hmm, the implementation seems to not accept a default value so this was reflected by the reference source generator. I'm not sure whether the discrepancy is a something that needs fixing, so I'll just revert this.